### DISCUSSION TITLE
Autobot questions

### DISCUSSION THREAD

**Discussion by mercuree** [musescore/MuseScore#19452] [Q&A]
I'm trying to make some automation using autobot and ran into problems which don't know how to workaround:
* Unable to emulate creation of local key/time signature, which usually requires use of CTRL+Click on palette cell. I can use `api.shortcuts.activate()` but there is no such shortcut to emulate CTRL+Click. 
* Unable to access measure properties for selected measure. Calling `api.dispatcher.dispatch("measure-properties")` opens dialog with wrong context (probably, note or rest, not measure itself)

Appreciate any help

**Comment by SadeeshaJayaweera** [comment]


**Reply by cbjeukendrup** [reply]
That looks suspiciously like AI-generated-nonsense... And I wonder why you're suddenly commenting here with such a comment while it seems you have never interacted with anything of MuseScore before.
Excuse me if I'm too distrustful but we gotta be alert for spam these days.
